File: lab01/zad02.m
A = 2.5;
ts = 0.005;
T = 4;
f = 2;

t = 0 : ts : T - ts;
y = sin(2 * pi * f * t) * A;
ys = randn(size(y)) * 0.1  + y;

plot(t,y)
hold on
plot(t, ys)
hold off

File: lab01/zad02.py
import numpy as np
import matplotlib.pyplot as plt

A = 2.5
ts = 0.005
T = 4
f = 2

t = np.arange(0., T, ts)
y = np.sin(2 * np.pi * f * t) * A
ys = y + 0.1 * np.random.randn(*y.shape)

fig, ax = plt.subplots()
ax.plot(t, y)
ax.plot(t, ys)
plt.show()
